Remotion/Validation/UnitTests/Validators/DecimalValidatorTest.cs

```diff
@@ -134,8 +134,8 @@ public void Validate_WithPropertyValueDecimalPlacesGreaterThanValidatorValue_Ret
 
       Assert.That(validationFailures.Length, Is.EqualTo(1));
       Assert.That(validationFailures[0].ValidatedObject, Is.EqualTo(propertyValidatorContext.Instance));
-      Assert.That(validationFailures[0].ValidatedProperty, Is.EqualTo(propertyValidatorContext.Property));
-      Assert.That(validationFailures[0].ValidatedPropertyValue, Is.EqualTo(propertyValidatorContext.PropertyValue));
+      Assert.That(validationFailures[0].ValidatedProperties.Single().Property, Is.EqualTo(propertyValidatorContext.Property));
+      Assert.That(validationFailures[0].ValidatedProperties.Single().ValidatedPropertyValue, Is.EqualTo(propertyValidatorContext.PropertyValue));
```

Comment:
Efficient way to write a test for this...
option 1:
```
Assert.That(validationFailures[0].ValidatedProperties.Select(vp => vp.Property), Is.EqualTo(new [] { propertyValidatorContext.Property }));
Assert.That(validationFailures[0].ValidatedProperties.Select(vp => vp.ValidatedPropertyValue), Is.EqualTo(new [] { propertyValidatorContext.PropertyValue }));
```

option 2:
```
Assert.That(validationFailures[0].ValidatedProperties.Select(vp => (vp.Property, vp.ValidatedPropertyValue), Is.EqualTo(new [] { (propertyValidatorContext.Property, propertyValidatorContext.ValidatedPropertyValue) }));
```

option 3:
```
Assert.That(validationFailures[0].ValidatedProperties.Length, Is.EqualTo(1));
Assert.That(validationFailures[0].ValidatedProperties, Has.One.Property(nameof(ValidatedProperty.Property)).EqualTo(propertyValidatorContext.Property));
Assert.That(validationFailures[0].ValidatedProperties, Has.One.Property(nameof(ValidatedProperty.ValidatedPropertyValue)).EqualTo(propertyValidatorContext.ValidatedPropertyValue));
```